tests/test_match_counter.py

```diff
@@ -59,7 +59,7 @@ def test_matched(self):
         self.assertCountEqual(counter.matched_keys(pkg), ['url', 'summary'])
         self.assertCountEqual(counter.matched_haystacks(pkg), [url, summary])
 
-    @mock.patch('dnf.match_counter.MatchCounter._eval_distance', return_value=0)
+    @mock.patch('dnf.match_counter.MatchCounter._eval_weights', return_value=0)
```

Comment:
Why do you mock this function? I'd say it's a core part of the sorting algorithm and should be part of the tests?